sonic-net/sonic-buildimage
# [Supermicro]: Enhancement for SSE-T7132S

#### Why I did it
1. Fix bugs for SONiC MGMT test cases.
2. Add more hwskus for multiple port mappings.
##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
1. Fix the bugs about platform APIs.
2. Add the new hwskus into device/supermicro/x86_64-supermicro_sse_t7132s-r0/.
#### How to verify it
1. Build a Innovium target for SSE-T7132S.

2. Run this image on SSE-T7132S and verify all dockers are up and test basic commands like:
show version
show platform summary
show platform syseeprom
show platform fan
show platform psustatus
show platform firmware status
show platform temperature
show platform ssdhealth

3. Verify ports are up using "show interface status" command

4. Run sonic-mgmt PTF test to verify the switch


#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
